recipes/common/Makefile.common

```diff
@@ -62,11 +62,6 @@ MISTRAL_MODEL_URL := https://huggingface.co/TheBloke/Mistral-7B-Instruct-v0.1-GG
 
 MODEL_NAME ?= $(MISTRAL_MODEL_NAME)
 
-.PHONY: download-model-mistral
-download-model-mistral:
-	cd ../../../models && \
-	make MODEL_NAME=mistral-7b-instruct-v0.1.Q4_K_M.gguf MODEL_URL=https://huggingface.co/TheBloke/Mistral-7B-Instruct-v0.1-GGUF/resolve/main/mistral-7b-instruct-v0.1.Q4_K_M.gguf -f  Makefile download-model
```

Comment:
I think this should be:
```Makefile
.PHONY: download-model-mistral
download-model-mistral:
	$(MAKE) -f  ../../../models/Makefile download-model-mistral
```

Reply:
I guess the question is where do we want the models downloaded to, if we do this I think the models will be downloaded in the model_servers or recipes directory?  Is that the correct place to download them.

I was surprised to find all of these huge files scattered in subdirs. Where I would be less surprised to find them in the models subdir.

Reply:
cd ../../../modles; $(MAKE) download-model-mistral

Would always download in the models dir.

Reply:
The purpose of the models dir, is to house all the models. There should not be models anywhere else in the repo.